src/main/java/jp/co/future/uroborosql/filter/SecretColumnSqlFilter.java

```diff
@@ -123,7 +127,8 @@ public void initialize() {
 			store = KeyStore.getInstance("JCEKS");
 
 			char[] pass;
-			try (InputStream is = new BufferedInputStream(new FileInputStream(storeFile))) {
+			try (InputStream fis = Files.newInputStream(storeFile);
+					InputStream is = new BufferedInputStream(fis)) {
```

Comment:
InputStream is = new BufferedInputStream(Files.newInputStream(storeFile)) is good?